inception-project/inception
# Issue #4166: Retain xml:id of TEI sentences

**What's in the PR**
- Backport of https://github.com/dkpro/dkpro-core/issues/1570

**How to test manually**
* Try importing a TEI document that has `xml:id` attributes on its sentences and see their IDs in the brat editor view

**Automatic testing**

**Documentation**
